front/src/routes/integration/all/netatmo/setting/EmptyState.jsx

```diff
@@ -0,0 +1,13 @@
+import { MarkupText } from 'preact-i18n';
+import cx from 'classnames';
+import style from './style.css';
+
+const EmptyState = () => (
+  <div class="col-md-12">
+    <div class={cx('text-center', style.emptyStateDivBox)}>
+      <MarkupText id="integration.tasmota.discover.noDeviceFound" />
```

Comment:
Ok, I think we should share words in a common dictionnary. Here you use tasmota key.

Reply:
Hi !! 
Thank you @atrovato for your review.  Well indeed all these points are not normal, it was supposed to be modified and corrected on an old merge.  I think there was a problem between our different merges. The files are fine with me and have no errors.  Sorry for the time wasted.  I correct and merge / push again tomorrow.

Reply:
Ok sorry, files deleted here : https://github.com/GladysAssistant/Gladys/pull/1015/commits/cd9f2252a960c4fbab8cb3212590c55bc8603496